docs/index.md

```diff
@@ -1,97 +1,114 @@
 ## Overview
-This guide walk-through the process of deploying an Sample Java application in CFT environment.
 
-This guide uses a pre-configured application from a template. Application is deployed to a kubernetes cluster. 
-At the end of this tutorial, you should be able to access an working application via VPN and made changes to it.
+This guide will walk you through the process of deploying a sample Java application in CFT.
 
-## Prerequisite
+An application will be generated from a template and deployed to a kubernetes cluster.
 
-`Github` access is required to complete the steps in this tutorial. Following the [link](https://hmcts.github.io/onboarding/team/github.html#github) to request access.
+At the end of this tutorial, you will be able to access your application via VPN and will have made changes to it.
 
-## High Level Steps
+## Prerequisites
+
+GitHub access is required to complete the steps in this tutorial. See the [onboarding guide](https://hmcts.github.io/onboarding/team/github.html#github) to get setup.
 
-- Create a Repository and build pipeline
-- Build application
-- Configure load balancing for HA
-- Deploy application
-- Access application
-- Customize application
-- Troubleshooting
 
 ## Steps
 
 #### Create a Repository and build pipeline
 
-1. Access [Backstage](https://backstage.platform.hmcts.net/create) to create a new github repo by selecting predefined [`Spring Boot Service`](https://backstage.platform.hmcts.net/create/templates/springboot-template) template. Please make sure name of  Github repo  starts with `labs*`. This enables Jenkins build tool to organize the folder based on Github repo names.
-
-2. Create Helm values file with following contents(this is only needed in the lab as the application will be run on sandbox).
+1. Click `Create` in the Backstage sidebar and select [`Spring Boot Service`](https://backstage.platform.hmcts.net/create) template. Prefix your GitHub repository name with `labs-*`. This enables Jenkins to automatically pickup the folder based on the GitHub repository name.
```

Comment:
no point telling them to prefix when you're telling them how to fill out the fields so you can drop that sentence